target/linux/ath79/dts/ar7100_mikrotik_routerboard-rb4xx.dtsi

```diff
@@ -0,0 +1,154 @@
+// SPDX-License-Identifier: GPL-1.0-or-later OR MIT
+
+// Based on ar71xx arch/mips/ath79/mach-rb4xx.c
+
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar7100.dtsi"
+
+/ {
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	aliases {
+		led-boot = &led_user_yellow;
+		led-failsafe = &led_user_yellow;
+		led-running = &led_user_yellow;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_user_yellow: user_yellow {
+			label = "rb4xx:yellow:user";
```

Comment:
first part of label should either be model name or vendor. I think we used "mikrotik" before, but I'm not sure.

If you use model name, this has to be moved to DTS.

Reply:
I'll change them to "mikrotik" as per the other mikrotik board dts checked in